### PR TITLE
Sql diagram

### PR DESCRIPTION
**Pull request by Fayob**
- add patients table
- add medical_histories table
- add treatments table
- add medical_treatment
- add invoices
- add invoice_items

### CODE REVIEW

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Fayob  and @amon-cofie  :wave: ,
I am Nemwel your reviewer for this PR :man_beard: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## :heavy_multiplication_x:  Invalid Code Review Request :heavy_multiplication_x: 

As this is a pair programming task, kindly ensure that all three partners make a meaningful contributions to the project (Including David Amon-Cofie)

Also, kindly ensure that you use the new Microverse README template

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Fayob** [comment]
@Nemwel-Boniface, this repository was created using provided template from microverse and the readme file was the one everyone used for the project

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+);
+
+CREATE TABLE invoice_items (
+	id SERIAL PRIMARY KEY,
+  unit_price DECIMAL,
+  quantity INT,
+  total_price DECIMAL,
+  invoice_id INT REFERENCES invoices(id),
+	treatment_id INT REFERENCES treatments(id)
+);
```
Comment (Nemwel-Boniface) [inline schema_based_on_diagram.sql]: - [ ] You have all done a great job adding also the join table as well as the foreign keys correctly. The only thing that is left is adding indexes for all the foreign keys that you have. **Hint:** I will create one for you ```CREATE INDEX ON invoices (medical_history_id);```

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @Fayob , @amon-cofie  and @FaranoshAmini            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Thank you for addressing the issues I raised earlier :+1:  and great job working on your project so far! :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -10,21 +84,168 @@ This repository includes files with plain SQL that can be used to recreate a dat
...
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (1GORDON) [inline README.md:247]: Great job team 🤝

- [ ] [Optionaal] Kindly fix the broken MIT license link to make your readme file more professional.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @Fayob & @amon-cofie & @FaranoshAmini ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**